Tests/AutoKeyboardNavigatorTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AutoKeyboardNavigatorTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import XCTest
+@testable import KeyboardSupport
+
+class AutoKeyboardNavigatorTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var keyboardNavigator: KeyboardAutoNavigator?
+    private var delegateMock: MockKeyboardAutoNavigatorDelegate?
+    
+    // MARK: - Tests
+    
+    override func setUp() {
+        super.setUp()
+        delegateMock = MockKeyboardAutoNavigatorDelegate()
+        keyboardNavigator = KeyboardAutoNavigator(navigationContainer: UIView(), defaultToolbar: KeyboardToolbar(), returnKeyNavigationEnabled: true)
+        keyboardNavigator?.delegate = delegateMock
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        keyboardNavigator = nil
```

Comment:
I know we recreate the delegate in `setUp()`, but we may want to `nil` it out here just to be consistent.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/b30ef2048cc3b6625c9a1e0dcbe9ecc4cbdbde47